### PR TITLE
feat: Location ref data extend LRD DB (RDCC-2644)

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-2644

### Change description ###
feat: Location ref data extend LRD DB (RDCC-2644)

**Does this PR introduce a breaking change?** (check one with "x")
No

### CODE REVIEW

File: src/integrationTest/resources/db/migration/V1_9__create_tables.sql [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	region_id varchar(16),
+	description varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT region_id_pk PRIMARY KEY (region_id)
+);
+
+create table building_location(
+	building_location_id varchar(16),
+	epimms_id integer,
```
Comment (kavithagade12) [inline src/integrationTest/resources/db/migration/V1_9__create_tables.sql:11]: Please change the epics_id to varchar(16)

File: src/integrationTest/resources/db/migration/V1_9__create_tables.sql [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+	status varchar(32),
+	welsh_status varchar(16),
+	created_time timestamp,
+	updated_time timestamp,
+	constraint building_location_status_id_pk primary key (building_location_status_id)
+);
+
+create table court_location(
+	court_location_id varchar(16),
+	epimms_id integer,
```
Comment (kavithagade12) [inline src/integrationTest/resources/db/migration/V1_9__create_tables.sql:106]: Please change epics_id to archer(16). the same is agreed with Srini.

File: src/integrationTest/resources/db/migration/V1_9__create_tables.sql [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+	closed_date timestamp,
+	court_location_code varchar(8),
+	dx_address varchar(16),
+	welsh_court_location_name varchar(256),
+	welsh_court_address varchar(512),
+	constraint court_id_pk primary key (court_location_id),
+	constraint court_location_code_uq unique (court_location_code)
+);
+
+create table judicial_location_mapping(
```
Comment (abhijitdiwan) [inline src/integrationTest/resources/db/migration/V1_9__create_tables.sql:126]: judicial_location_mapping no need of this. we will need to add this in JRD remove this.

File: src/integrationTest/resources/db/migration/V1_9__create_tables.sql [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+alter table court_district_civil_jurisdiction_assoc add CONSTRAINT dcj_court_location_id_fk
+FOREIGN KEY (court_location_id)
+REFERENCES COURT_LOCATION (court_location_id);
+
+alter table court_district_civil_jurisdiction_assoc add CONSTRAINT dcj_court_district_civil_jurisdiction_id_fk
+FOREIGN KEY (district_civil_jurisdiction_id)
+REFERENCES DISTRICT_CIVIL_JURISDICTION (district_civil_jurisdiction_id);
+
+alter table JUDICIAL_LOCATION_MAPPING add CONSTRAINT judicial_location_mapping_epimms_id_fk
```
Comment (abhijitdiwan) [inline src/integrationTest/resources/db/migration/V1_9__create_tables.sql:180]: remove this also JUDICIAL_LOCATION_MAPPING

File: src/integrationTest/resources/db/migration/V1_9__create_tables.sql [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+	court_location_name varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	region_id varchar(16),
+	court_location_category_id varchar(16),
+	cluster_id varchar(16),
+	open_for_public boolean,
+	court_address varchar(512),
+	postcode varchar(8),
+	phone_number varchar(16),
```
Comment (abhijitdiwan) [inline src/integrationTest/resources/db/migration/V1_9__create_tables.sql:116]: phono varchar(128)

File: src/integrationTest/resources/db/migration/V1_9__create_tables.sql (hunk 3 above)
Comment (kavithagade12) [reply src/integrationTest/resources/db/migration/V1_9__create_tables.sql:126]: Agreed please delete this table we will create the same in jrd

File: src/main/resources/db/migration/V1_9__create_tables.sql [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+alter table court_district_civil_jurisdiction_assoc add CONSTRAINT dcj_court_location_id_fk
+FOREIGN KEY (court_location_id)
+REFERENCES COURT_LOCATION (court_location_id);
+
+alter table court_district_civil_jurisdiction_assoc add CONSTRAINT dcj_court_district_civil_jurisdiction_id_fk
+FOREIGN KEY (district_civil_jurisdiction_id)
+REFERENCES DISTRICT_CIVIL_JURISDICTION (district_civil_jurisdiction_id);
+
+alter table JUDICIAL_LOCATION_MAPPING add CONSTRAINT judicial_location_mapping_epimms_id_fk
+FOREIGN KEY (epimms_id)
```
Comment (kavithagade12) [inline src/main/resources/db/migration/V1_9__create_tables.sql:181]: Please remove all the judicial location mapping references

**Review by kavithagade12 — APPROVED** [review]

**Comment by SujeethKumarAri** [comment]
Closing as changes are being addressed in https://github.com/hmcts/rd-location-ref-api/pull/209

**Pull request closed without merging**